crawl-ref/source/job-data.h

```diff
@@ -77,6 +77,17 @@ static const map<job_type, job_def> job_data =
       { SK_THROWING, 2 }, { SK_WEAPON, 2 }, },
 } },
 
+{ JOB_BATTLEMAGE, {
+    "Ba", "Battlemage",
+    4, 4, 4,
+    { SP_TENGU, SP_KOBOLD, SP_BASE_DRACONIAN, SP_NAGA, SP_TROLL,
+      SP_MERFOLK, SP_DEMONSPAWN, },
+    { "leather armour plus:1", "book of War Wizardry" },
+    WCHOICE_NONE,
+    { { SK_FIGHTING, 1}, { SK_UNARMED_COMBAT, 3 }, { SK_DODGING, 2 },
+      { SK_SPELLCASTING, 2 }, { SK_TRANSMUTATIONS, 1 }, { SK_HEXES, 1 }, },
```

Comment:
If anything, it would be better to consolidate more book backgrounds than to add new ones. Backgrounds that combine elementalists would be rad af, e.g. fire/earth and ice/air. Similarly, combining necromancer and summoner into a single ally-oriented demonologist (or whatever) background. I could see a "battlemage" as a replacement for both skald and wizard. Perhaps a witch/warlock background that combines direct damage necromancy with hexes could work as a new enchanter. 

Book backgrounds that lend themselves to several different paths of development from the very beginning would help distinguish hellcrawl from mainline crawl. Lots of "design space" to look at there.